front_end_ang/src/app/app.module.ts

```diff
@@ -2,7 +2,7 @@ import { NgModule } from '@angular/core';
 import { BrowserModule, provideClientHydration } from '@angular/platform-browser';
 import { FormsModule } from '@angular/forms';
 import { ReactiveFormsModule } from '@angular/forms';
-import { HttpClientModule } from '@angular/common/http';
+import { HttpClientModule, provideHttpClient, withFetch } from '@angular/common/http';
 import { MatInputModule } from '@angular/material/input';
```

Comment:
esta alteração foi pra resolver esse warning abaixo que aparecia?

NG02801: Angular detected that `HttpClient` is not configured to use `fetch` APIs. It's strongly recommended to enable `fetch` for applications that use Server-Side Rendering for better performance and compatibility. To enable `fetch`, add the `withFetch()` to the `provideHttpClient()` call at the root of the application.

Reply:
acredito fortemente que sim, zeh

Reply:
sim exatamente

Reply:
Não é preciso fazer alterações em nenhuma parte do código? Me lembro que quando estava fazendo um teste para isso tinha alguns erros que pararam de serem exibidos corretamente com o ToastrModule.

Mas pode ser também porque eu tenha feito algo indevido quando fiz

Reply:
Cara imagino que não, até pq as chamadas pra api localmente tavam funcionando normal. Vc sabe me dizer como reproduzir esse cenario do toast com erro?

Reply:
Um exemplo é tentar deletar um cliente que está envolvido em uma movimentação de saída

Reply:
1. Não tem toaster no cenário de erro, é um alert só
2. O js do client lança uma exception pq ele tenta serializar a response do endpoint como json mas é enviado uma string
3. Cenário de sucesso tem toast funcionando